readme.md

```diff
@@ -0,0 +1,39 @@
+# City Temperature Management API
+This project is built on FastAPI and provides functionalities related to managing cities and temperature records.
+
+To clone this project from GitHub, follow these steps:
+1. Open your terminal or command prompt.
+2. Navigate to the directory where you want to clone the project.
+3. Run the following command:
+```shell
+git clone https://github.com/ArturPoltser/py-fastapi-city-temperature-management-api.git

```

Comment:
Why in your instruction you give link for clonning repository of some `ArturPoltser`?